app/src/pages/inside/common/accordion/useAccordionTabsState.js

```diff
@@ -26,7 +26,7 @@ export const useAccordionTabsState = (initialState) => {
       (acc, id) => ({ ...acc, [id]: currentTab === id }),
       {},
     );
-    setTabsState(newTabsState);
+    currentTab && setTabsState(newTabsState);
```

Comment:
you don't needed modify this hook. To avoid collapsing by clicking options just remove `collapseTabsExceptCurr` from `OptionsBlock` component